.github/workflows/main.yml

```diff
@@ -189,3 +189,16 @@ jobs:
         uses: isort/isort-action@master
         with:
             configuration: "--profile black"
+  test_poetry:
```

Comment:
what do you think about including this with the lint configuration?  it's mostly similar, and conceptually i think it belongs there.

Reply:
sure

Reply:
changed it